src/interactive.rs

```diff
@@ -198,6 +202,80 @@ fn handle_choice_input(
     }
 }
 
+// simple function so we can easily get more complicated later if we need to
+fn parse_list(provided_value: &str) -> Vec<String> {
+    provided_value.split(',').map(|s| s.to_string()).collect()
```

Comment:
for now I would put the choices separator `','` into a constant. 
So that it is easier to have it also configurable (via cli / env argument) in the future.